src/agent/src/storage/fs_handler.rs

```diff
@@ -50,6 +50,23 @@ impl StorageHandler for OverlayfsHandler {
                 .options
                 .push(format!("workdir={}", work.to_string_lossy()));
         }
+        storage
+            .driver_options
+            .iter()
+            .filter_map(|e| {
+                e.as_str()
+                    .strip_prefix("io.katacontainers.volume.overlayfs.create_directory=")
```

Comment:
How about defining a const in kata-types/volume.rs for `io.katacontainers.volume.overlayfs.create_directory`?

Reply:
Done.